--- lib/devices/android/chrome.js
+++ lib/devices/android/chrome.js
@@ -106,7 +107,8 @@
 ChromeAndroid.prototype.prepareChromedriver = function (cb) {
   var chromeArgs = {
     port: this.args.proxyPort,
-    executable: this.args.chromedriverExecutable
+    executable: this.args.chromedriverExecutable,
+    adbPort: (process.env.ANDROID_ADB_SERVER_PORT || 5037)
   };
   this.chromedriver = new Chromedriver(chromeArgs);
   this.proxyReqRes = this.chromedriver.proxyReq.bind(this.chromedriver);
